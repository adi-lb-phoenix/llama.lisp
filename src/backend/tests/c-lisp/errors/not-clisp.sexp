(brilisp
    (define ((main int))
        (ret 0)))
