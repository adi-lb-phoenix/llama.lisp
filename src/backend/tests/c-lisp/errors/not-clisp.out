CodegenError: Input not a C-Lisp program
